ml-agents/mlagents/trainers/policy/torch_policy.py

```diff
@@ -164,19 +161,17 @@ def evaluate(
         :param decision_requests: DecisionStep object containing inputs.
         :return: Outputs from network as defined by self.inference_dict.
         """
-        vec_vis_obs, masks = self._split_decision_step(decision_requests)
-        vec_obs = [torch.as_tensor(vec_vis_obs.vector_observations)]
-        vis_obs = [
-            torch.as_tensor(vis_ob) for vis_ob in vec_vis_obs.visual_observations
-        ]
+        obs, masks = self._split_decision_step(decision_requests)
+        obs = [torch.as_tensor(np_ob) for np_ob in obs]
```

Comment:
maybe `np_obs` and `tensor_obs` to make the types clearer (instead of redefining `obs`)?